tests/test_local_python_executor.py

```diff
@@ -122,6 +122,24 @@ def test_evaluate_f_string(self):
         assert result == "This is x: 3."
         self.assertDictEqualNoPrint(state, {"x": 3, "text": "This is x: 3.", "_operations_count": 6})
 
+    def test_evaluate_f_string_with_format(self):
+        code = "text = f'This is x: {x:.2f}.'"
+        state = {"x": 3.336}
+        result, _ = evaluate_python_code(code, {}, state=state)
+        # evaluate returns the value of the last assignment.
```

Comment:
```suggestion
```

Reply:
thank you! completely forgotten to do so after adding a second test